shared/uavobjectdefinition/loggingsettings.xml

```diff
@@ -1,6 +1,7 @@
 <xml>
 	<object name="LoggingSettings" singleinstance="true" settings="true">
 		<description>Settings for the logging module</description>
+		<field name="LogDestination" units="" type="enum" options="SPIFlash,OpenLog" elements="1" defaultvalue="OpenLog"/>
```

Comment:
Wouldn't it have been better to leave the default `SPIFlash`? Also, as far as I can tell it crashes when `OpenLog` is set but the port is not setup for this.